src/app/frontend/_variables.scss

```diff
@@ -24,6 +24,7 @@ $headline-font-size-base:  rem(2.4) !default;
 $subhead-font-size-base:   rem(1.6) !default;
 $body-font-size-base:      rem(1.4) !default;
 $caption-font-size-base:   rem(1.2) !default;
+$toolbar-height-size-base: rem(6.6) !default;
```

Comment:
Can this be a multiplier of $baseline-grid? Maybe 64px? 2px less is nothign and the toolbar is already large.